src/main/java/frc/robot/Ports.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot;
+
+public final class Ports {
+    public static final class Conveyor {
+        public static final int MOTOR = 0;
+        public static final boolean IS_MOTOR_INVERTED = false;
+        public static final int SHOOTER_PROXIMITY = 0;
```

Comment:
```suggestion
        public static final int PROXIMITY_SENSOR = 0;
```
Why is the "shooter" needed?
Also, don't forget it's a sensor. And BTW, aren't you using a laser sensor?

Reply:
We use Laser sensor, but I don't know how to call an object of this "type"

Reply:
`LASER_SENSOR` is good.
However, I just realized that if you really use a laser sensor, you'll need to change some of the code because these are digital and not analog like proximity sensors.

Reply:
@orel-david says that is analog

Reply:
😢
That's unfortunate. I strongly recommend using laser beam sensors, they are simpler to work with and more accurate IMO.
Anyway, if you are using a proximity sensor, why https://github.com/Galaxia5987/robot-2021/commit/5cb1adaa02fe495155a7d02475ca36c6fb01ee63?

Reply:
We are using the Laser Beam, We are going to connect it (somehow, don't ask me how) to an analog sensor

Reply:
There is some confusion here. Usually when talking about "laser beam sensors" you talk about a pair of items, where one launches a laser beam and the other receives it ([example](https://www.aliexpress.com/item/4000060455626.html)). The output of the sensor is **digital**: "0" if the sensor doesn't receive the beam, "1" otherwise. So when an object passes through the beam and blocks it, you can know that.
A "proximity sensor" is a sensor that consists of one item that launches a laser beam and receives its reflection. Based on the [time of flight](https://en.wikipedia.org/wiki/Time_of_flight) (ToF), it outputs a varying voltage which is proportional to the time it took the beam to return to the sensor. This sensor is **analog** because it outputs a range of values and not just a pair of them like a digital sensor.

So which one are you using in the conveyor?
> somehow, don't ask me how

I think it's good knowledge to have – you program the subsystem, you should know how it works.

Reply:
I know how laser beam and proximity sensor works, I told you that they are planning to buy something that is equivalent in the correctness of the laser beam (even-though most likely it won't happen) ,or they somehow going to connect it to an "translator"

Reply:
You see, I like writing texts.
> I know how laser beam and proximity sensor works

So although I realized that while I was writing the explanation, I wanted to finish it anyway 😝.
> I told you [...] something that is equivalent in the correctness of the laser beam

I'm not sure what that means, but your [comment](https://github.com/Galaxia5987/robot-2021/pull/9#discussion_r566653975) doesn't mention anything like this.
> I'm sorry, but I still don't understand what type the sensor is – analog or digital?

Reply:
I told you the second thing, and for your 2nd question, just ask @orel-david, he can explain it better

Reply:
Sure, I'll wait for his response. But I think it's important you'll know how your subsystem works.

Reply:
We havn't bought the sensor yet and some of the sensors we looked at were analog, so for now it is fine if the code refers to them as analog ,because even if they are digital they would return 5v or 0 so the deadband approach would still work

Reply:
I see.
> because even if they are digital they would return 5v or 0 so the deadband approach would still work

True, but it will be simpler if you treated the sensor as digital.